node/forest_libp2p/src/service.rs

```diff
@@ -259,7 +259,8 @@ where
                     None => { break; }
                 },
                 interval_event = interval.next() => if interval_event.is_some() {
-                    info!("Peers connected: {}", swarm_stream.get_ref().peers().len());
+                    //info!("Peers connected: {}", swarm_stream.get_ref().peers().len());
```

Comment:
Delete comment

Reply:
This just shouldn't be commented out (imo, if anyone disagrees please say why)

Reply:
I'd say uncomment it.